sgv2-restapi/pom.xml

```diff
@@ -259,6 +259,20 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
       </plugin>
+      <!-- Plugin for building Docker images -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>3.1.4</version>
+        <configuration>
+          <from>
+            <image>adoptopenjdk:8</image>
```

Comment:
this image is not the same as the one in the `Dockerfile`

Reply:
ignore the dockerfile, it will be deleted